cloud/helpers.js

```diff
@@ -0,0 +1,126 @@
+/*
+  Computes a unit reward value in Wei using the formula:
+  toWei(maxReward) / maxTaskers - 1
+*/
+async function computeUnitRewardWei(maxReward, maxTaskers) {
+  const maxRewardWei = Number(
+    await Moralis.Cloud.units({
+      method: "toWei",
+      value: maxReward,
+    })
+  );
+  const unitRewardWei = maxRewardWei / maxTaskers - 1; // ensures that unitRewardWei*maxTaskers < maxRewardWei
```

Comment:
could this potentially be the source of the reward discrepancy between the MoralisDB and smart contract? Seems like unitRewardWei is rounded down